src/Momento.ts

```diff
@@ -115,9 +118,78 @@ export class Momento {
     });
   }
 
+  public async createSigningKey(
+    ttlMinutes: number,
+    endpoint: string
+  ): Promise<CreateSigningKeyResponse> {
+    this.validateTtlMinutes(ttlMinutes);
+    const request = new control.control_client._CreateSigningKeyRequest();
+    request.ttl_minutes = ttlMinutes;
+    return await new Promise<CreateSigningKeyResponse>((resolve, reject) => {
+      this.client.CreateSigningKey(
+        request,
+        {interceptors: this.interceptors},
+        (err, resp) => {
+          if (err) {
+            reject(cacheServiceErrorMapper(err));
+          } else {
+            resolve(new CreateSigningKeyResponse(endpoint, resp));
+          }
+        }
+      );
+    });
+  }
+
+  public async revokeSigningKey(
+    keyId: string
+  ): Promise<RevokeSigningKeyResponse> {
+    const request = new control.control_client._RevokeSigningKeyRequest();
+    request.key_id = keyId;
+    return await new Promise<RevokeSigningKeyResponse>((resolve, reject) => {
+      this.client.RevokeSigningKey(
+        request,
+        {interceptors: this.interceptors},
+        err => {
+          if (err) {
+            reject(cacheServiceErrorMapper(err));
+          } else {
+            resolve(new RevokeSigningKeyResponse());
+          }
+        }
+      );
+    });
+  }
+
+  public async listSigningKeys(
+    endpoint: string,
+    nextToken?: string
+  ): Promise<ListSigningKeysResponse> {
+    const request = new control.control_client._ListSigningKeysRequest();
+    request.next_token = nextToken ?? '';
+    return await new Promise<ListSigningKeysResponse>((resolve, reject) => {
+      this.client.ListSigningKeys(
+        request,
+        {interceptors: this.interceptors},
+        (err, resp) => {
+          if (err) {
+            reject(cacheServiceErrorMapper(err));
+          } else {
+            resolve(new ListSigningKeysResponse(endpoint, resp));
+          }
+        }
+      );
+    });
+  }
+
   private validateCacheName = (name: string) => {
     if (!name.trim()) {
       throw new InvalidArgumentError('cache name must not be empty');
     }
   };
+
+  private validateTtlMinutes = (ttlMinutes: number) => {
+    if (ttlMinutes && ttlMinutes < 0) {
+      throw new InvalidArgumentError('ttlMinutes must be positive');
+    }
+  };
```

Comment:
Honestly this was just to keep with the existing pattern, happy to move this to a standard utility function declaration

Reply:
Every other function in this file is a normal function. The `validateCacheName` function property is the one that is nonconforming. Instead of adding to the nonconformity maybe fix it (since it's a small cosmetic change next to stuff you're already changing) and do your thing the way the majority of the file does things?